main.go

```diff
@@ -344,23 +337,29 @@ func main() {
 			failf("Failed to copy build artifact, error: %s", err)
 		}
 
-		isSigned, err := isBuildArtifactSigned(aapt, unsignedBuildArtifactPth)
-		if err != nil {
-			failf("Failed to check if build artifact is signed, error: %s", err)
-		}
+		signAAB := strings.EqualFold(artifactExt, ".aab")
+
+		if signAAB || !cfg.UseAPKSigner {
```

Comment:
Why is the check for `signAAB` needed here, is not `!cfg.UseAPKSigner` enough?

Reply:
Well hopefully nobody selects apksigner for aab but it can be done theoretically and it will use jarsigner anyway. This is making sure we ignore the apksigner flag if it is aab

Reply:
I see.